node/package.json

```diff
@@ -20,7 +20,11 @@
   "homepage": "https://github.com/iopipe/examples#readme",
   "dependencies": {
     "@iopipe/iopipe": "^1.5.0",
+    "@types/aws-lambda": "^8.10.17",
+    "@types/iopipe__iopipe": "^1.12.0",
```

Comment:
We also control the `iopipe` package. Are we able to do something like `@types/iopipe` as well?

Reply:
Yup we can add something to types/iopipe entry if that makes sense. But that only really makes sense if we had a library that could be installed at `iopipe` (non namespaced). I think `iopipe` is deprecated and points to `@iopipe/iopipe`.

Reply:
Yes, i is "deprecated", but we're still publishing releases to it. So if that's what it takes to get `@types/iopipe` then maybe we just continue to support it?